src/components/RequestModal/TvRequestModal.tsx

```diff
@@ -637,7 +632,7 @@ const TvRequestModal = ({
                           </td>
                           <td className="whitespace-nowrap px-1 py-4 text-sm font-medium leading-5 text-gray-100 md:px-6">
                             {season.seasonNumber === 0
-                              ? intl.formatMessage(messages.extras)
+                              ? intl.formatMessage(messages.specials)
```

Comment:
If the season number is 1 won't we still need `intl.formatMessage(messages.extras)`?

Reply:
I'm not sure I understand @westdom 

- Firstly, I refactored the variable from "Extras" to "Specials" because typically, Season 0 of a TV Show is typically referred to as "Specials". So by the change I made in line 44 of this file, it deprecates the usage of Extras entirely.
- Secondly, if the season number is 1 in this ternary operator, then it would use the code in line 636 of this file; specifically the message `messages.seasonnumber`. So, Extras would not play a role there either.